web/src/pages/Settings/Streams/StreamModal/StreamModal.jsx

```diff
@@ -228,6 +217,20 @@ const StreamModal = ({ stream, onClose, apply, del }) => {
                                     }}
                                 />
                             );
+                        default:
```

Comment:
I don't think I like falling through to a `default` like this. We should handle two types - text, and password.